# Customized statesharing message
uint64 timestamp
uint64 timestamp_real_time
uint16 agent_id

# global position coordinates
float32 global_position_lon
float32 global_position_lat
float32 global_position_alt

# quaternion
float32[4] q

# TOPICS incoming_state_sharing outgoing_state_sharing
